core/templates/dev/head/domain/collection/CollectionNodeObjectFactorySpec.js

```diff
@@ -103,8 +103,7 @@ describe('Collection node object factory', function() {
     expect(prerequisiteSkillList.getSkillCount()).toEqual(0);
     expect(acquiredSkillList.getSkillCount()).toEqual(0);
     expect(collectionNode.getExplorationSummaryObject()).toEqual({
-      exists: true,
-      newlyCreated: true
+      exists: true
```

Comment:
I'm starting to get confused about what collectionNodeBackendObject.exploration is supposed to contain. Here it looks rather sparse, but from _initializeExplorationDetails I assumed it would be a full summary. I think the keys should be consistent for all these objects.